packages/node_modules/@webex/plugin-meetings/src/meeting/index.js

```diff
@@ -1386,6 +1386,10 @@ export default class Meeting extends StatelessWebexPlugin {
 
       this.mediaProperties.remoteStream = streams;
 
+      // Set all stream(s) quality once remote stream's bytes are recieved
+      // This will do nothing if override value isn't given
+      this.setMeetingQuality(this.mediaProperties.qualityLevel);
```

Comment:
I'm not sure why this is needed. Isn't the meeting quality already set at this point?

Reply:
I thought this was the agreed spot to set the quality, since calling `setMeetingQuality` on join causes failures due to the remote stream not existing to set it's quality?
So, like the comment says, when the remote stream's bytes are received then set the meeting's quality with the given `meetingQuality` value set at join

Reply:
It seems like we should be able to set the stream quality at the time of peer connection establishment instead of having to get a standard peer connection at first and then renegotiate a different quality one. 
@arun3528  thoughts?

Reply:
agreed it should be part of addMedia